lambda/app.js

```diff
@@ -1,43 +1,48 @@
 'use strict';
 
-var dxf = require('dxf'),
-  request = require('request');
+const dxf = require('dxf');
+const request = require('request');
 
-exports.handler = function (event, context) {
+exports.handler = (event, context) => {
   const body = JSON.parse(event.body);
 
-  request({ url: body.url, encoding: null }, function (error, response, body) {
-    var dxfContents = body.toString();
-    var units = determineUnits(dxfContents);
-    var svgSTR = dxf.toSVG(dxf.parseString(dxfContents));
-    svgSTR = svgSTR.replace("INSUNITS", units);
-    svgSTR = svgSTR.replace("INSUNITS", units);
-    context.done(null, { svg: svgSTR });
+  request({ url: body.url, encoding: null }, (error, response, body) => {
+    const dxfContents = body.toString();
+
+    const outputSVG = dxf
+      .toSVG(dxf.parseString(dxfContents))
+      .replace("INSUNITS", determineUnits(dxfContents));
+
+    context.done(
+      null,
+      {
+        isBase64Encoded: false,
+        statusCode: 200,
+        headers: { "Access-Control-Allow-Origin": "*" },
+        body: JSON.stringify({ "svg": outputSVG })
+      }
+    );
   });
 };
 
 function determineUnits(dxfContents) {
-  var unitSVG = "INSUNITS";
+  let unitSVG = "INSUNITS";
```

Comment:
:ribbon: would be nice to move this to the top of the file so we can reference the same constant inside `handler`.

Reply:
Good call, refactored in https://github.com/inventables/dxfimport/pull/4/commits/88cbbfe1a89ff389c109916928aef0964e6307de